api/api.go

```diff
@@ -4,18 +4,27 @@ import (
 	"errors"
 	"fmt"
 	"net/http"
+	"regexp"
 	"time"
 
 	"github.com/labstack/echo/v4"
 	"github.com/nuts-foundation/nuts-auth/pkg"
 	"github.com/sirupsen/logrus"
 )
 
-// Wrapper bridges the generated api types and http logic to the internal types and logic
+// Wrapper bridges the generated api types and http logic to the internal types and logic.
+// It checks required parameters and message body. It converts data from api to internal types.
+// Then passes the internal formats to the AuthClient. Converts internal results back to the generated
+// Api types. Handles errors and returns the correct http response. It does not perform any business logic.
 type Wrapper struct {
 	Auth pkg.AuthClient
 }
 
+// TODO when the registry allows
+var vendorIdentifierFromHeader = func(ctx echo.Context) string {
```

Comment:
Is the use of this header temporary? Requesting vendor ID should be accessible from SSL client certificate. Or is a reverse proxy adding this for you? If so, it might be better to add the certificate as header, since getting the SubjectAltName from a certificate requires custom logic?

Reply:
Yes, a reverse proxy should add this. The idea to add the complete certificate is a good one. Any suggestions for a header name which includes the client certificate?

Reply:
Copy the exact name from nginx: $ssl_client_escaped_cert -> X-Ssl-Client-Escaped-Cert

See: http://nginx.org/en/docs/http/ngx_http_ssl_module.html